Cargo.toml

```
@@ -29,6 +29,7 @@ toml = { version = "0.8.8" }
 tokio = { version = "1.34.0", features = ["macros", "rt"] }
 prefixed-api-key = { version = "0.1.0", features = ["sha2"]}
 prettytable-rs = { version = "0.10.0"}
+urlencoding = "2.1.3"
```

We can standardize in another PR, but just keeping consistent with the previous lines
```suggestion
urlencoding = { version = "2.1.3" }
```